modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1834,6 +1857,103 @@ private void _testPostObjectDefinitionBatch() throws Exception {
 		Assert.assertEquals(2, page.getTotalCount());
 	}
 
+	@Test
+	private void _testPostRelatedSystemModifiableObjectDefinitionBatch()
```

Comment:
@JhossephG rename to `_testPostObjectDefinitionWithSystemAggregationObjectField();`